common/Model/Api.php

```diff
@@ -181,13 +182,18 @@ public function sendInvitation(Order $order) {
         return $this->postInvitation($request, $config);
     }
 
-    private function getProducts(array $orderItems, array $config, ?int $storeId): array {
-        if (empty($config['product_reviews'])) {
+    private function getProductsSafe(Collection $orderItems, array $config, ?int $storeId): array {
+        try {
+            return $this->getProducts($orderItems, $config, $storeId);
+        } catch (\Exception $e) {
+            $this->logger->debug(sprintf('Error retrieving products: %s', $e->getMessage()));
```

Comment:
Would suggest logging the class and code of the exception.

```suggestion
            $this->logger->debug(sprintf('Error retrieving products: %s: (%s) %s', get_class($e), $e->getCode(), $e->getMessage()));
```

Reply:
👍